pkg/kubelet/kubelet.go

```diff
@@ -381,9 +392,30 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		}
 	}
 
+	var nodeHasSynced cache.InformerSynced
+	var nodeLister corelisters.NodeLister
+
+	if kubeDeps.KubeClient != nil {
```

Comment:
Since this code block got moved up, the comment on old L427/new L459 is now after the first instance of this pattern, might wanna move that up.

https://github.com/kubernetes/kubernetes/blob/3c514ae588fdd2867eb9cced1c43c9ee2276a7ce/pkg/kubelet/kubelet.go#L426-L427

Reply:
makes sense, will update.